<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWin</class>
 <widget class="QMainWindow" name="PlotWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plots</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QwtPlot" name="qwtPlot"/>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="xypBox">
        <property name="frame">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>X</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Phi</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="pvaBox">
        <item>
         <property name="text">
          <string>Position</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Velocity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Acceleration</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PlotWin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>764</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>712</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
